player/cache.go

```diff
@@ -220,6 +251,11 @@ func (c *fsCache) Set(hash string, body []byte) (ReadableChunk, error) {
 	return &cachedChunk{reflectedChunk{body}}, nil
 }
 
+// set adds the entry in the cache. returns true if successful, false if unsuccessful
+func (c *fsCache) set(hash string, cacheCost int64) bool {
```

Comment:
Other methods are public, I'd leave them at that

Reply:
the choice of making it private is so that it can't be misused from the outside. I think it's best if it remains private, or?